streams/src/main/java/org/apache/kafka/streams/processor/WallclockTimestampExtractor.java

```diff
@@ -28,6 +28,13 @@
  * built-in <i>CreateTime</i> timestamp (see KIP-32: Add timestamps to Kafka message for details).
  */
 public class WallclockTimestampExtractor implements TimestampExtractor {
+
+    /**
+     * Return the current wall clock time as timestamp.
+     *
+     * @param record  a message
```

Comment:
Note: In the Streams terminology, we say "record" (or "data record").  We don't use the term "message" (we say "message" in Kafka).  This terminology change needs to be done throughout the PR.